bouncer/src/repo/manipulator/repo_manipulator.cpp

```diff
@@ -71,6 +71,25 @@ bool RepoManipulator::connectAndAuthenticateWithAdmin(
 	return handler != 0;
 }
 
+bool RepoManipulator::connectAndAuthenticateWithAdmin(
+	std::string       &errMsg,
+	const std::string &connString,
+	const uint32_t    &maxConnections,
+	const std::string &username,
+	const std::string &password,
+	const bool        &pwDigested
+)
+{
+	//FIXME: we should have a database manager class that will instantiate new handlers/give existing handlers
```

Comment:
I guess this will be tackled in a separate issue

Reply:
Doesn't `getHandler` do this job?

Reply:
this is about a possibility of having 2 database connection to 2 different places.

it's a bit N/A as that's only a concern with 3drepogui, which is deprecated for years